models/marts/core/marketing/dim_customers.sql

```diff
@@ -0,0 +1,99 @@
+with customers as (
+    select * from {{ ref('stg_ecommerce__customers') }}
+    ),
+
+orders as (
+    select * from {{ ref('stg_ecommerce__orders') }}  
+    ),
+
+order_items as (
+    select * from {{ ref('stg_ecommerce__order_items') }} 
+    ),
+
+order_reviews as (
+    select * from {{ ref('stg_ecommerce__order_reviews') }}   
+    ),
+
+cancelled_missing_orders as (
+    select
+    distinct order_id
+    from    orders
```

Comment:
for readability it's best to keep spacing/alignment consistent. Generally we recommend just one space after the `from`